### ISSUE TITLE
Document how to use these toolchains

### ISSUE THREAD

**Issue by aiuto** [bazelbuild/java_tools#30]
This snippet was suggested for the bazel  3.5 release notes. It should be really be with the 14.0 release description.

```
# Java tools javac14 for Darwin
http_archive(
    name = "remote_java_tools_darwin",
    sha256 = "e20f002ceb3f3353d64c022e1f3400d8539ee56ffcfd4a6680d73d6a2cac938b",
    urls = [
        "https://mirror.bazel.build/bazel_java_tools/releases/javac14/v1.0/java_tools_javac14_darwin-v1.0.zip",
        "https://github.com/bazelbuild/java_tools/releases/download/javac14-v1.0/java_tools_javac14_darwin-v1.0.zip",
    ],
)

# Zulu OpenJDK for Darwin
http_archive(
    name = "openjdk14_darwin_archive",
    build_file_content = "java_runtime(name = 'runtime', srcs =  glob(['**']), visibility = ['//visibility:public'])",
    sha256 = "088bd4d0890acc9f032b738283bf0f26b2a55c50b02d1c8a12c451d8ddf080dd",
    strip_prefix = "zulu14.28.21-ca-jdk14.0.1-macosx_x64",
    urls = ["https://mirror.bazel.build/cdn.azul.com/zulu/bin/zulu14.28.21-ca-jdk14.0.1-macosx_x64.tar.gz"],
)
```
And invoke the build with these parameters:
```
$ bazel build \
     --java_toolchain=@remote_java_tools_darwin//:toolchain_jdk_14 \
     --host_java_toolchain=@remote_java_tools_darwin//:toolchain_jdk_14 \
     --javabase=@openjdk14_darwin_archive//:runtime \
     --host_javabase=@openjdk14_darwin_archive//:runtime \
   :foo
```

**Comment by aiuto** [comment]
cc: @davido

**Comment by davido** [comment]
@aiuto 

> This snippet was suggested for the bazel 3.5 release notes. It should be really be with the 14.0 release description.

I think you meant: "It should be really be with the 3.4 release description."

In fact, the above snippet would work just fine with Bazel 3.4, because of Error Prone version was bumped to very recent version in Bazel 3.4 that supported new java JDKs.

However, `java_tools_javac14_darwin-v1.0.zip` release was only published after Bazel release 3.4 was conducted.
And the new `toolchain_jdk_14` was based on this [PR](https://github.com/bazelbuild/bazel/pull/11514) that was not included in Bazel 3.4 release.

Bottom line: we should clarify where and how to announce new Java toolchains releases.

**Comment by aiuto** [comment]
The complaint in the review of the release notes is that the code snippet
does not belong in the blog post.

> Bottom line: we should clarify where and how to announce new Java
toolchains releases.
Agreed.
In general, announcements of new toolchains should be with the repository
that provides them.
Instructions for using them as well.
Further, I want to see release notes for incremental (non-major) releases
be fully automated, so there is no need to get PR authors involved in the
text review.  Then we only do wordsmithing on major releases, where we do
the highlights and point elsewhere for usage instructions.
